### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'laravel_version')

### ISSUE THREAD

**Issue by iammati** [spatie/ignition#133]
**Please include some context and the contents of the console in your browser's developer tools.**

![image](https://user-images.githubusercontent.com/41418763/168489940-c1d706fb-d86e-40ac-83e6-15a03a56891f.png)

I'm running `spatie/ignition` in TYPO3 CMS which worked just perfectly fine. I've performt an update of few packages via composer, will try to undo the update and comment here the results.

## JavaScript Error
```
TypeError: Cannot read properties of undefined (reading 'laravel_version')
    at https://ze.ddev.site/:174:282166
    at kv (https://ze.ddev.site/:174:282562)
    at Ef (https://ze.ddev.site/:170:186618)
    at https://ze.ddev.site/:170:221150
    at vp (https://ze.ddev.site/:170:222502)
    at HTMLUnknownElement.v (https://ze.ddev.site/:170:103604)
    at Object.Bn (https://ze.ddev.site/:170:103929)
    at qn (https://ze.ddev.site/:170:104890)
    at xh (https://ze.ddev.site/:170:270689)
    at uh (https://ze.ddev.site/:170:261914)
```

## Reproduction Steps
Please tell us what you were doing when this error occurred, so we can more easily debug it and find a solution.

I was actually just writing some PHP code (see below) - when the error/exception occurred - and used `symfony/var-dumper` `dd()` function:
```php
$normalizedParams = $request->getAttribute('normalizedParams');
$uri = $normalizedParams->getRequestUri();
dd($uri);
```

## User Agent
```
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.67 Safari/537.36
```

**Comment by iammati** [comment]
![image](https://user-images.githubusercontent.com/41418763/168490058-0a76a86b-16a5-4740-b66f-75200d95b45f.png)

Yup, downgrading back from 1.3.0 to 1.2.10 everything works just fine and I can see again the exception handler rendering the page.

**Comment by casanasvictor** [comment]
Yes i opened the same issue and also i needed to revert my composer.json and also i had the same result.

**Comment by AlexVanderbist** [comment]
Hi @iammati, @casanasvictor,

Sorry about that! Looks like we were a bit too quick with testing and tagging a new version of our UI library. I have a possible fix in https://github.com/spatie/ignition/releases/tag/1.3.1. Can either of you try updating your spatie/ignition dependency to see if the error is fixed? 

You can do this using `composer require spatie/ignition` (or `composer require spatie/laravel-ignition -W` when using Laravel).

Thanks

**Comment by iammati** [comment]
@AlexVanderbist yes it just works fine. The error is no longer occurring in v1.3.1, thanks! (:

**Comment by casanasvictor** [comment]
@AlexVanderbist i can confirm this fix works just fine, thank you for taking care of this
